openapi/spec.go

```diff
@@ -191,6 +193,7 @@ type Operation struct {
 	Responses   Responses         `json:"responses,omitempty" yaml:"responses,omitempty"`
 	Deprecated  bool              `json:"deprecated,omitempty" yaml:"deprecated,omitempty"`
 	Servers     []*Server         `json:"servers,omitempty" yaml:"servers,omitempty"`
+	Security    []*Security       `json:"security,omitempty" yaml:"security,omitempty"`
```

Comment:
According to the spec, it should be a Security Requirement Object, which seems to be a non-type field, i.e `map[string]interface{}`.